packages/webapp-libs/webapp-crud-demo/src/hooks/useCrudDemoPaginatioQuery/useCrudDemoPaginationQuery.hook.ts

```diff
@@ -0,0 +1,53 @@
+import { useQuery } from '@apollo/client';
+import { useEffect, useState } from 'react';
+import { crudDemoItemListQuery } from '@sb/webapp-crud-demo/routes/crudDemoItem/crudDemoItemList/crudDemoItemList.component';
+
+export const useCrudDemoPaginationQuery = (options?: any) => {
+  const [cachedCursrors, setCachedCursors] = useState<Array<string>>([]);
```

Comment:
Do you need this? I thought that this could be just sth like:
```
const [currentPage, setCurrentPage] = useState<number>(1)
```
and based on the incrementing / decrementing this state return hasPrevious